### PR TITLE
Tailor

### PR DESCRIPTION
**Pull request by RiceGodAC**
Tailor

### CODE REVIEW

File: sql/weenies/PK Trophy Vendor/4200098 Enhanced Robe of the Tundra.sql [hunk 1]
```diff
@@ -0,0 +1,46 @@
+DELETE FROM `weenie` WHERE `class_Id` = 4200098;
+
+INSERT INTO `weenie` (`class_Id`, `class_Name`, `type`, `last_Modified`)
+VALUES (4200098, 'ace4200098-enhancedrobeofthetundratailorr', 2, '2021-11-01 00:00:00') /* Clothing */;
+
+INSERT INTO `weenie_properties_int` (`object_Id`, `type`, `value`)
+VALUES (4200098,   1,          2) /* ItemType - Clothing */
+     , (4200098,   3,          9) /* PaletteTemplate - Grey */
+     , (4200098,   4,       1024) /* ClothingPriority - OuterwearUpperLegs, OuterwearLowerLegs, OuterwearChest, OuterwearAbdomen, OuterwearUpperArms, OuterwearLowerArms, Head, Feet */
```
Comment (Advan-tage) [inline sql/weenies/PK Trophy Vendor/4200098 Enhanced Robe of the Tundra.sql:9]: Dont you want this to be Chest Coverage?

Comment (Advan-tage) [reply sql/weenies/PK Trophy Vendor/4200098 Enhanced Robe of the Tundra.sql:9]: nevermind it is chest coverage. you just didnt change the comments

File: sql/weenies/PK Trophy Vendor/4200101 Turquoise Winged Helmet.sql [hunk 2]
```diff
@@ -0,0 +1,46 @@
+DELETE FROM `weenie` WHERE `class_Id` = 4200101;
+
+INSERT INTO `weenie` (`class_Id`, `class_Name`, `type`, `last_Modified`)
+VALUES (4200101, 'helmetgromniewingedtailor', 2, '2005-02-09 10:00:00') /* Clothing */;
+
+INSERT INTO `weenie_properties_int` (`object_Id`, `type`, `value`)
+VALUES (4200101,   1,          2) /* ItemType - Armor */
+     , (4200101,   3,          1) /* PaletteTemplate - AquaBlue */
+     , (4200101,   4,          1) /* ClothingPriority - Head */
```
Comment (Advan-tage) [inline sql/weenies/PK Trophy Vendor/4200101 Turquoise Winged Helmet.sql]: clothing priority for head should be 16384

Comment (RiceGodAC) [reply sql/weenies/PK Trophy Vendor/4200101 Turquoise Winged Helmet.sql]: submitted the change

**Comment by RiceGodAC** [comment]
I didn't update most or all of the description to the changes I've made in the 1st commit but in the recent ones with the 2h weapons I've done it for all the lines I've changed as I understand it'll make it much easier for you to review it. Sorry for the inconveniences caused in the first commit!

**Pull request closed without merging**